src/components/AddBareMetalHosts/BareMetalInventoryAddHost.tsx

```diff
@@ -42,8 +43,9 @@ const BaremetalInventoryAddHosts: React.FC = () => {
           <HostsNotShowingLink setDiscoveryHintModalOpen={setDiscoveryHintModalOpen} />
         </Text>
         <HostRequirements ContentComponent={HostRequirementsContent} />
+        <FormatDiskWarning />
+        <VMRebootConfigurationInfo hosts={cluster.hosts} />
       </TextContent>
-      {/* <FormatDiskWarning /> */}
```

Comment:
@jtomasek Please confirm this should be already re-enabled

Reply:
Yes, it should be enabled.